packages/grafana-ui/src/components/Table/TableCellInspectModal.tsx

```diff
@@ -60,15 +59,8 @@ export function TableCellInspectModal({ value, onDismiss, mode }: TableCellInspe
         <pre>{text}</pre>
       )}
       <Modal.ButtonRow>
-        <ClipboardButton getText={() => text} onClipboardCopy={() => setIsInClipboard(true)}>
-          {!isInClipboard ? (
-            'Copy to Clipboard'
-          ) : (
-            <>
-              <Icon name="check" />
-              Copied to clipboard
-            </>
-          )}
+        <ClipboardButton icon="check" getText={() => text} onClipboardCopy={() => setIsInClipboard(true)}>
```

Comment:
I don't think this needs the `onClipboardCopy={() => setIsInClipboard(true)}` and more (and the associated code from it) as that was doing the same "Copied!" stuff that you've added in the base component